src/rgw/rgw_admin.cc

```diff
@@ -74,6 +74,7 @@ void _usage()
   cout << "  bucket rm                  remove bucket\n";
   cout << "  bucket check               check bucket index\n";
   cout << "  bucket reshard             reshard bucket\n";
+  cout << "  bucket fixmpleak           remove leaked multipart objects that have arisen from retries\n";
```

Comment:
@wschroeder How good to use `remove leaked multipart objects those piled from retries` in place of `remove leaked multipart objects that have arisen from retries`

Reply:
I had similar feedback from some others, and I am `s/arisen/accumulated/`.